docs/x509/reference.rst

```diff
@@ -2592,11 +2592,10 @@ These classes may be present within a :class:`CertificatePolicies` instance.
 
         :type: list
 
-        A list consisting of :term:`text` and/or :class:`UserNotice` objects.
-        If the value is text it is a pointer to the practice statement
-        published by the certificate authority. If it is a user notice it is
-        meant for display to the relying party when the certificate is
-        used.
+        A list consisting of ``str`` and/or :class:`UserNotice` objects.  If the
+        value is text it is a pointer to the practice statement published by the
```

Comment:
```suggestion
        value is ``str`` it is a pointer to the practice statement published by the
```

Reply:
(Didn’t check line length on this recommendation)

Reply:
I made that change and rewrapped.